SS2-Project/Assets/Starstorm2/Modules/Pickups/Equipments/AffixPurple.cs

```diff
@@ -145,7 +146,7 @@ public void OnDamageDealtServer(DamageReport damageReport)
 
             private void FixedUpdate()
             {
-                if (NetworkServer.active)
+                if (HasAnyStacks && NetworkServer.active)
```

Comment:
HasAnyStacks is unecesary. Again, this check only works for the Interfaces that apply to the behaviour. FixedUpdate is not called when the behaviour is disabled (AKA: when Stacks <= 0)